CoreUtilities/src/au/gov/asd/tac/constellation/utilities/geospatial/Shape.java

```diff
@@ -526,6 +527,10 @@ public static void generateGeoPackage(final String uuid, final Map<String, Strin
         featureEntry.setBounds(ReferencedEnvelope.EVERYTHING);
         featureEntry.setSrid(spatialReference.getSrid());
 
+        // Remove geopackage file if exists as it stores existing tables/indexes
+        if (output.isFile()) {
+            Files.delete(output.toPath());
```

Comment:
Wouldn't a simple output.delete(); work here?

Reply:
While it does work to use `output.delete()` here, the recommendation from SonarQube is to use `Files.delete()`, apparently for better error reporting, reliability and maintainability.